DFC.Digital/DFC.Digital.Service.CourseSearch/Converters/MessageConverter.cs

```diff
@@ -85,13 +92,24 @@ internal static CourseDetails ConvertToCourseDetails(this CourseDetailOutput api
                 AssessmentMethod = apiCourseDetail.Course.AssessmentMethod,
                 EquipmentRequired = apiCourseDetail.Course.EquipmentRequired,
                 QualificationName = apiCourseDetail.Course.QualificationTitle,
-                Cost = apiCourseDetail.Opportunity.FirstOrDefault()?.Price,
-                VenueDetails = new VenueDetails
+                QualificationLevel = apiCourseDetail.Course.QualificationLevel,
+                VenueDetails = new Venue
                 {
                     EmailAddress = apiCourseDetail.Venue.FirstOrDefault()?.Email,
```

Comment:
Provide reason comment for using "FirstOrDefault"

Reply:
done